style.css

```diff
@@ -0,0 +1,8 @@
+#image {
+    width: 100%;
+    height: auto;
+    max-width: 625px;
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
+    }
```

Comment:
Always make sure that the index.html semantic tag is styled individually for a better result.